legwork/psd.py

```diff
@@ -143,14 +154,23 @@ def Sc(f, t_obs):
     else:
         R = load_response_function(f, fstar)
 
+    # if sky position and polarization are specified, include them
+    # instead of the averaged response; to do this, we need to undo
+    # the 10/3 averaging term
+    if (theta is not None) & (phi is not None) & (psi is not None):
+        R = 3 / 10 * F_plus_squared(theta, phi, psi)
+    elif (theta is not None) & (phi is not None) & (psi is None):
+        psi = np.random.uniform(0, 2 * np.pi)
```

Comment:
I think this should have a warning about the assumed random value. Hmm though not sure how to deal with the "double" warnings if this gets called from source.py.